package.json

```diff
@@ -11,14 +11,15 @@
   ],
   "main": "riemann",
   "engines": {
-    "node": "> 0.6.0"
+    "node": ">= 0.12"
   },
   "repository": {
     "type": "git",
     "url": "git://github.com/perezd/riemann-nodejs-client.git"
   },
   "dependencies": {
-    "node-protobuf": "^1.4.3"
+    "node-protobuf": "^1.4.3",
+    "event-to-promise": "^0.8.0"
```

Comment:
Have you reviewed this dependency closely? What to ensure we really need to take on an additional dep for our customers. Could this be inlined?

Reply:
This one is the most popular among similar libs, the most 'alive' on Github, has tests and has no any additional dependencies.
Actually it consists of a single file and we could repeat its functionality but I'm not sure if it's a good thing to do.

Reply:
SGTM.

Reply:
So, are we going to leave this dependency as is OR to copy-paste the code?
I vote for the first option.